src/components/tag/Tag.stories.mdx

```diff
@@ -1,40 +1,82 @@
 import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
 import { Tag } from ".";
-import { GnuiContainer } from "../container";
+import { Heading } from "../heading";
+import { Flex } from "../container";
+import { Breadcrumbs } from "../breadcrumbs";
 
 <Meta title="Components/Tag" component={Tag} />
 
 # Tag
 
+- `color`: `string` – _optional_, background color of Tag by default background is `theme.colors.lights[2]`
```

Comment:
background color of Tag, DEFAULT: `theme.colors.lights[2]

I think more concise -> better when it comes to documenting props